examples/use_two_plugins.py

```diff
@@ -0,0 +1,272 @@
+                    await room.delete(room_member)
+                else:
+                    await msg.say('登录用户不是该群管理员...')
+            else:
+                await msg.say(f'can not fine room member by name<{room_member_name}>')
+        elif text.startswith('get room topic'):
+            logger.info('get room topic')
+            if room:
+                topic: Optional[str] = await room.topic()
+                if topic:
+                    await msg.say(topic)
+
+        elif text.startswith('rename room topic:'):
+            logger.info('rename room topic ...')
+            if room:
+                new_topic = text[len('rename room topic:') + 1:]
+                await msg.say(new_topic)
+        elif text.startswith('add new friend:'):
+            logger.info('add new friendship ...')
+            identity_info = text[len('add new friend:') + 1:]
+            weixin_contact: Optional[Contact] = await self.Friendship.search(weixin=identity_info)
+            phone_contact: Optional[Contact] = await self.Friendship.search(phone=identity_info)
+            contact: Optional[Contact] = weixin_contact or phone_contact
+            if contact:
+                self.Friendship.add(contact, 'hello world ...')
+
+        elif text.startswith('at me'):
+            await msg.say(self.login_user)
+
+        else:
+            pass
+
+
+    async def on_friendship(self, friendship: Friendship):
+        """
+        处理好友请求
+        """
+        administrator = bot.Contact.load('admin-id')
+        await administrator.ready()
+
+        contact = friendship.contact()
+        await contact.ready()
+
+        log_msg = f' {contact.name} 请求加好友'
+        await administrator.say(log_msg)
+
+        if friendship.type() == FriendshipType.FRIENDSHIP_TYPE_RECEIVE:
+            if friendship.hello():# == 'ding':
+                # log_msg = 'accepted automatically because verify messsage is "ding"'
+                log_msg = '自动通过好友'
+                await asyncio.sleep(2)
+                print('等待通过好友 ...')
+                await friendship.accept()
+                # if want to send msg, you need to delay sometimes
+
+                print('等待发送欢迎消息 ...')
+                await asyncio.sleep(3)
+                await contact.say('你好')
+                await contact.say('如果知道入群关键词，可以直接发我关键词哟')
+                print('已经通过好友 ...')
+            else:
+                log_msg = 'not auto accepted, because verify message is: ' + friendship.hello()
+
+        elif friendship.type() == FriendshipType.FRIENDSHIP_TYPE_CONFIRM:
+            log_msg = 'friend ship confirmed with ' + contact.name
+
+        print(log_msg)
+        await administrator.say(log_msg)
+
+ 
+    async def on_login(self, contact: Contact):
+        """login event
+        Args:
+            contact (Contact): the account logined
+        """
+        logger.info('微信号<%s> 已经登录……', contact)
+        self.login_user = contact
```

Comment:
we should mainly add the plugin usage code not the usages of wechaty interface.  Please remove it and add more plugin example code.